### ISSUE TITLE
Add Picture Taking/Sharing Functionality

### ISSUE THREAD

**Issue by Corbzilla** [Prodigious-Aim-Solutions/inroad#13]
Add the ability for a user to take a picture and share it socially.

**Comment by Corbzilla** [comment]
Open this issue for further discussion on picture taking/sharing.

So after a little research, we've got a couple of solutions:
- Instagram
  - can't upload directly to Instagram though API.
  - Large user base, growing fast
  - can easily get pics and sort by hash tags
- Flickr
  - can upload directly to Flickr through API
  - smaller user base, not as well known
  - getting and sorting seem to be as easy as Instagram

So basically one we can upload the pic directly to the service, the other we can't. Instagram has more users and is growing fast, we can still get pics from the service but uploading would consist of re-directing them out of our web app to the Instagram app (web/native). Flickr we can do the uploading directly but loose the user base/awareness.

Thoughts @AshleyCB @kiefer999 ?

**Comment by Corbzilla** [comment]
The more I think about this, the less I'm concerned with the upload functionality.

A lot of the parks won't have wi-fi or cell signal, so as long as the can take the picture, save locally and organize it there for upload/sharing later, I think that might be OK.

**Comment by kiefer999** [comment]
```
                                                                                  It looks like Flickr is more doable, do you have to have the app reviewed by them first?                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        From: Jason CorbettSent: Friday, March 13, 2015 3:33 PMTo: Prodigious-Aim-Solutions/inroadReply To: Prodigious-Aim-Solutions/inroadCc: kiefer999Subject: Re: [inroad] Add Picture Taking/Sharing Functionality (#13)Open this issue for further discussion on picture taking/sharing.
```

So after a little research, we've got a couple of solutions:

Instagram

can't upload directly to Instagram though API.
Large user base, growing fast
can easily get pics and sort by hash tags

Flickr

can upload directly to Flickr through API
smaller user base, not as well known
getting and sorting seem to be as easy as Instagram

So basically one we can upload the pic directly to the service, the other we can't. Instagram has more users and is growing fast, we can still get pics from the service but uploading would consist of re-directing them out of our web app to the Instagram app (web/native). Flickr we can do the uploading directly but loose the user base/awareness.

Thoughts @AshleyCB @kiefer999 ?

—Reply to this email directly or view it on GitHub.

**Comment by Corbzilla** [comment]
Nope, no review, you can actually get all the public photos just by signing up but for people to post their photos they'd have to have a Flickr account which is why I'm leaning towards just saving the photos locally and adding a few links to launch the native app/navigate to the Web site and let them share themselves
